src/Drupal/Commands/core/BatchCommands.php

```diff
@@ -17,7 +17,7 @@ class BatchCommands extends DrushCommands
      *
      * @return \Consolidation\OutputFormatters\StructuredData\UnstructuredListData
      */
-    public function process($batch_id, $options = ['format' => 'json'])
+    public function process($batch_id, $options = ['format' => 'null'])
```

Comment:
The caller expects JSON to be returned. See https://github.com/drush-ops/drush/blob/92b2359f8d448adba5fcf1e0d2ecd1584c8c7d73/includes/batch.inc#L152-L153